01week/datatypes.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+// 1. program for date and time
+function returnDate() {
+  const now = new Date();
+  return now;
+}
+console.log(returnDate());
+
+// 2. convert a number to a string
+const num = 1578;
+const n = num.toString();
```

Comment:
This needs to be in a function. A function that accepts a parameter and converts that number to a string and returns it.